devel/cluster/kind_cluster_node_versions.sh

```diff
@@ -14,18 +14,18 @@
 
 # generated by ./hack/latest-kind-images.sh
 
-KIND_IMAGE_K8S_120=docker.io/kindest/node@sha256:d67de8f84143adebe80a07672f370365ec7d23f93dc86866f0e29fa29ce026fe
-KIND_IMAGE_K8S_121=docker.io/kindest/node@sha256:f9b4d3d1112f24a7254d2ee296f177f628f9b4c1b32f0006567af11b91c1f301
+KIND_IMAGE_K8S_120=docker.io/kindest/node@sha256:45d0194a8069c46483a0e509088ab9249302af561ebee76a1281a1f08ecb4ed3
+KIND_IMAGE_K8S_121=docker.io/kindest/node@sha256:ad5b7446dd8332439f22a1efdac73670f0da158c00f0a70b45716e7ef3fae20b
 KIND_IMAGE_K8S_122=docker.io/kindest/node@sha256:bfd5eaae36849bfb3c1e3b9442f3da17d730718248939d9d547e86bbac5da586
```

Comment:
note: There's no real reason to update this as it'll likely be removed in #5523 but I figured I might as well update here since this PR will merge before that one!